libmproxy/console/__init__.py

```diff
@@ -225,6 +225,7 @@ def __init__(self, server, options):
                 if err:
                     print >> sys.stderr, "Script load error:", err
                     sys.exit(1)
+                script.ObserveScripts(self, i)
```

Comment:
We should initialize the observation in the constructor of the Script class - otherwise, we don't have this feature for mitmdump.